orchagent/aclorch.h

```diff
@@ -236,7 +261,36 @@ class AclRuleMirror: public AclRule
     MirrorOrch *m_pMirrorOrch;
 };
 
-class AclTable {
+class AclRuleDTelFlowWatchListEntry: public AclRule
+{
+public:
+    AclRuleDTelFlowWatchListEntry(AclOrch *m_pAclOrch, DTelOrch *m_pDTelOrch, string rule, string table, acl_table_type_t type);
+    bool validateAddAction(string attr_name, string attr_value);
+    bool validate();
+    bool create();
+    bool remove();
+    void update(SubjectType, void *);
+
+protected:
+    DTelOrch *m_pDTelOrch;
+    string m_intSessionId;
+    bool INT_enabled;
+    bool INT_session_valid;
+};
+
+class AclRuleDTelDropWatchListEntry: public AclRule
+{
+public:
+    AclRuleDTelDropWatchListEntry(AclOrch *m_pAclOrch, DTelOrch *m_pDTelOrch, string rule, string table, acl_table_type_t type);
+    bool validateAddAction(string attr_name, string attr_value);
+    bool validate();
+    void update(SubjectType, void *);
+
+protected:
+    DTelOrch *m_pDTelOrch;
+};
+
+struct AclTable {
```

Comment:
why change AclTable from class to struct?

Reply:
Fixed